genie-client/src/main/java/com/netflix/genie/client/apis/SortAttribute.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *
+ *  Copyright 2020 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.client.apis;
+
+/**
+ * Sort key for entity search results.
+ * This is a subset of all sort attributes supported by the server.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+public enum SortAttribute {
```

Comment:
FYI the server allows sorting by multiple attributes at once and this mechanism limits it to singular

Reply:
Also only allows sorting by 3 statically defined attributes. Seems like a good tradeoff in terms of convenience to me.
Anyone using this API doesn't have to think too hard because the enums make it impossible to misuse it.